src/test/java/com/wikia/webdriver/common/core/api/ForumBoardContent.java

```diff
@@ -1,9 +1,11 @@
 package com.wikia.webdriver.common.core.api;
 
 import com.wikia.webdriver.common.contentpatterns.URLsContent;
-import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.core.url.UrlBuilder;
+import org.apache.http.message.BasicNameValuePair;
+
+import java.util.ArrayList;
 
 import org.apache.http.message.BasicNameValuePair;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)